test/tools/ossfuzz/protoToAbiV2.h

```diff
@@ -0,0 +1,249 @@
+#pragma once
+
+#include <ostream>
+#include <sstream>
+#include <test/tools/ossfuzz/abiV2Proto.pb.h>
+#include <libdevcore/Whiskers.h>
+#include <libdevcore/FixedHash.h>
+
+namespace dev
+{
+namespace test
+{
+namespace abiv2fuzzer
+{
+class ProtoConverter
+{
+public:
+	ProtoConverter()
+	{
+		m_isStateVar = true;
+		m_counter = 0;
+		m_varCounter = 0;
+		// Return value on first error condition is 1,
+		// incremented for subsequent error conditions.
+		m_returnValue = 1;
+	}
+	ProtoConverter(ProtoConverter const&) = delete;
+	ProtoConverter(ProtoConverter&&) = delete;
+	std::string contractToString(Contract const& _input);
+
+private:
+	typedef std::vector<std::pair<bool,unsigned>> vecOfBoolUnsignedTy;
```

Comment:
`using VecOfBoolUnsignedTy = std::vector<std::pair<bool,unsigned>>`
(`using` and uppercase for type names)

Reply:
Oh and no need to append `Ty` if you use uppercase :)